scss/partials/_header.scss

```diff
@@ -3,11 +3,20 @@
  */
 .header {
   @include clearfix;
+  @include fixed( $left: $sidebar-width, $top: 0, $right: 0 );
```

Comment:
Any chance you could remove the space after `(` and the space before `)` perhaps? :)